docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-feature-integration.md

```diff
@@ -70,6 +70,7 @@ Adjust the schema definition so that entity changes will trigger events:
             <parameter name="spy_product_offer_product_offer_reference" column="product_offer_reference" keep-additional="true"/>
             <parameter name="spy_product_offer_concrete_sku" column="concrete_sku" keep-additional="true"/>
         </behavior>
+        <behavior name="\Spryker\Zed\AclEntity\Persistence\Propel\Behavior\AclEntityBehavior"/>
```

Comment:
seems this is related to ACL integration, lets remove

Reply:
cleaned up